aos/closed_loop/cutOutDonutsWithWEP.ipynb

```diff
@@ -8,10 +8,10 @@
     "# CutOutDonuts with WEP\n",
```

Comment:
<p>As in <code>calcZernikesFromDonutStamps</code> I think it would be good to mention the origin of the data, eg. that this update is not only updating <code>ts_wep</code> version, but also using <code>imsim</code> instead of <code>phosim</code> to simulate the donuts which explains different donut postage stamps.  If first notebook in the series  could point to the <code>closed loop</code> arguments used (or <code>imsim</code> config yaml file location)   that would be useful. I found the <code>wep</code> yaml <code>testCwfsPipelineConfig_v8.0.3.yaml</code> but I'm not sure what exact <code>imsim</code> config was used (to know whether eg. this had background or not, what was the source brightness, etc. )    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/39/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Added a data subsection under the Logistics section again that explains the image data came from imsim and the catalogs from <code>GenerateDonutCatalogWcsTask</code> explained in the <code>wepSourceSelectionWithWcs</code> notebook.  </p>

Reply:
<p>There is also a sample yaml at the end of every notebook in the sequence that shows how to generate the data.</p>